src/cmd/dist/buildgo.go

```diff
@@ -114,7 +113,7 @@ func mkzcgo(dir, file string) {
 	fmt.Fprintln(&buf)
 	fmt.Fprintf(&buf, "package build\n")
 	fmt.Fprintln(&buf)
-	fmt.Fprintf(&buf, "const defaultCGO_ENABLED = %q\n", os.Getenv("CGO_ENABLED"))
+	fmt.Fprintf(&buf, "const defaultCGO_ENABLED = %q\n", "1")
```

Comment:
How about just the empty string? So it'll use its normal heuristics

Reply:
Oh, didn't see it had extra behavior for unset. Done.